pallets/collator-assignment/src/benchmarking.rs

```diff
@@ -89,6 +91,9 @@ mod benchmarks {
             0
         );
 
+        // TODO: make sure this did not call assign_collators_rotate_all, because that's cheaper
```

Comment:
how can we make sure of this?

Reply:
I guess fetching previous collator assignment and the new one and make sure they are equal?

Reply:
It's not that easy because if the new assignment is random there is a small chance that it is equal to the output of the other function. I was thinking of adding an event or something.

Reply:
maybe we can add an event when randomness is [0:u32] as well and cover both things (smoke-test plus this?)

Reply:
Good idea, what about an event like:

```
// TODO: rename
CollatorAssignmentEvent {
    seed: [u8; 32],
    full_rotation: bool,
}
```

This way we can see the seed for debugging purposes, and also check that full_rotations happens when it should. Any more info that you think would be valuable? The session_index is already known, and for the collator assignment I believe there is another event in some other pallet.